source/guide_ejabberd.rst

```diff
@@ -29,100 +30,96 @@ ejabberd
   * :manual:`domains <web-domains>`
   * :manual:`firewall ports <basics-ports>`
   * :manual:`web backends <web-backends>`
+  * :manual:`web headers <web-headers>`
+  * :manual:`MySQL <database-mysql>`
 
 This guide is based on the initial `pull request <https://github.com/Uberspace/lab/pull/444>`_ from `fm0de <https://github.com/fm0de>`_ and the `work from clerie <https://blog.clerie.de/ejabberd-auf-uberspace-installieren/>`_.
 
-Prerequisites
-=============
-
-Domain
-------
-
-Your ejabberd domain ``isabell.example`` needs to be setup. ejabberd then defaults to the subomains
-``upload.isabell.example``, ``conference.isabell.example`` , ``pubsub.isabell.example`` and ``proxy.isabell.example``
-::
-
- [isabell@stardust ~]$ uberspace web domain add isabell.example
- [isabell@stardust ~]$ uberspace web domain add conference.isabell.example
- [isabell@stardust ~]$ uberspace web domain add upload.isabell.example
- [isabell@stardust ~]$ uberspace web domain add pubsub.isabell.example
- [isabell@stardust ~]$ uberspace web domain add proxy.isabell.example
-
-Also we need the SSL Certificates. Visit the domains in a browser or run:
-::
-
- [isabell@stardust ~]$ curl https://isabell.example
- [isabell@stardust ~]$ curl https://conference.isabell.example
- [isabell@stardust ~]$ curl https://upload.isabell.example
- [isabell@stardust ~]$ curl https://pubsub.isabell.example
- [isabell@stardust ~]$ curl https://proxy.isabell.example
-
 
 Installation
 ============
 
 Download, configure, compile and install ejabberd.
 Use the following options for ``./configure``:
 
-  * ``--prefix=$HOME/opt/ejabberd/``: Install to your personal uberspace
+  * ``--prefix=$HOME/``: Install to your personal uberspace
   * ``--enable-user=$USER``: Allow execution of ejabberd as $USER
-  * ``--enable-mysql --enable-new-sql-schema``: optionally compile with mysql support
-  * ``--enable-sqlite``: optionally compile with sqlite support
+  * ``--enable-mysql --enable-new-sql-schema``: optionally compile with mysql support and use the new schema
 
 Run ``./configure --help`` to see all options.
 
-::
+.. code-block:: console
 
- [isabell@stardust ~]$ mkdir $HOME/src/
- [isabell@stardust ~]$ cd $HOME/src/
- [isabell@stardust src]$ wget https://github.com/processone/ejabberd/archive/20.03.tar.gz
- [isabell@stardust src]$ tar xf 20.03.tar.gz
- [isabell@stardust src]$ cd ejabberd-20.03/
- [isabell@stardust ejabberd-20.03]$ ./autogen.sh
- [isabell@stardust ejabberd-20.03]$ ./configure --enable-user=$USER --prefix=$HOME/opt/ejabberd --enable-mysql --enable-new-sql-schema
- [isabell@stardust ejabberd-20.03]$ make install
+  [isabell@stardust ~]$ cd ~/downloads/
```

Comment:
There is no `~/downloads` directory on a standard uberspace. Please stick to `~/src` as it's more unixy or just drop the files directly in home.

Reply:
I changed `~/downloads` to the home directory `~/`.